### PR TITLE
Mint sign up form

### PR DESCRIPTION
**Pull request by TommyM0ntana**

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi team :wave: nice work and great improvement.

### However, there are few changes I would like us to implement :slightly_smiling_face:

- [x] Validate your html, [use this link](https://validator.w3.org/)

- [x] Make your [design](https://rawcdn.githack.com/TommyM0ntana/mint_singupform/0512619fd8efdd97b614acc264e88c086fac8ce6/signup.html) look like the original page [mint.com's signup page](https://accounts.intuit.com/signup.html) 
    * pointer :dart: ``` center your form```.

[**Take Note**] :point_down: 
 - [x] Make your **commit messages** more descriptive, for example refer to this commit message```e223115```. To find the ID, in your pull request click on commits, look through the list.

- [x] After implementing the changes as requested, **in your pull request, comment on the changes you made** :slightly_smiling_face: 

### Implement these changes and submit another form for code review.

### This project is not approved :stop_sign:

**Comment by tomzacchia** [comment]
@uche-inyama with reference to your suggestion for descriptive commit messages, we had an issue with VS Code and Github where tabs were somehow changed from 2 to 4 spaces when pushing code to the remote repo. As such we were testing different solutions to see what would work. In this case I think it would have made sense to make a separate branch to isolate these fixes since it was impossible to tell if what we were doing in vscode was going to fix this issues. Essentially we had to push multiple times to see if the Stickler issue could be resolved.

**Comment by tomzacchia** [comment]
- The form was centered on the page however the width of the parent of the intuit logo was smaller than the width of the image (child overflow), as such when you change the size of the screen it makes it look as if the form is not centered. The property of the parent's width was changed to fit-content to fix this UI bug.
- screenshot of HTML validation
Screen Shot 2019-09-19 at 11 48 16 AM

**Review by Forison — APPROVED** [review]
Good job Guys. 👋👋
Your project looks goods
### your project is approved.
## **[optional]** 
 > I would like to address the issue you raise about the width of the form.
by changing the form with the value from % to a pixel is the answer in this. for example, give the "FORM" in line 108 of the CSS width of 500px and see the changes you get.
Note > for now you have to assume you are building web pages for Desktop view. The time will come for you to design for different screen size

File: signup.css [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  margin-left: 1rem;
+  width: 1rem;
+}
+
+/* form */
+
+form {
+  margin: 0 auto;
+  margin-top: 1rem;
+  width: 75%;
```
Comment (Forison) [inline signup.css:108]: change this to 500px

**Pull request merged**